CVLByExample/ERC20/README.md

```diff
@@ -9,8 +9,27 @@ the following rules fail:
 `balancesBoundedByTotalSupply` - fails for functions `burn`, `deposit`, and `mint`.
 `totalSupplyIsSumOfBalances` - fails for functions `deposit` and `withdraw`.
 
+These rules fail for the following reasons:
+1. `deposit` updates `_balances[msg.sender]` but does not update `_totalSupply` accordingly.
+2. `withdraw updates `_balances[msg.sender]` but does not update `_totalSupply` accordingly.
+
+## Incorrect Spec
+1. The function `transferFrom(address,address,uint)` is defined in the method block of `ERC20.spec` as
+   `envfree` but it refers to `env` so cannot be defined as such.
+2. The invariant `balancesBoundedByTotalSupply` allows the receiver and sender in `transferFrom` to be the same.
```

Comment:
I don't understand - I think there is no problem in transferFrom? do you have a failure?

Reply:
Yes. envfreeFuncsStaticCheck fails on transferFrom.
https://prover.certora.com/output/1902/5e5037a8185c445aad5591d3f903b79d?anonymousKey=f8fec978de79db8556eda0248c120b344431857b

It uses env. It cannot be envfree.